pkg/virtctl/adm/logverbosity/logverbosity.go

```diff
@@ -290,35 +279,38 @@ func setVerbosity(currentLv map[string]uint, patchData *[]patch.PatchOperation,
 		JSONName := getJSONNameByComponentName(componentName)
 		currentLv[JSONName] = *verbosity
 	}
-
-	// in case of just reset (no set operation after the reset), don't need to add another patch
-	if len(currentLv) != 0 {
-		if !hasDeveloperConfiguration {
-			// if DeveloperConfiguration is absent, add DeveloperConfiguration first
-			addPatch(patchData, patchAdd, dcPath, &v1.DeveloperConfiguration{})
-		}
-		addPatch(patchData, patchAdd, lvPath, currentLv)
-	}
 }
 
 func createPatch(currentLv map[string]uint, hasDeveloperConfiguration bool) ([]byte, error) {
-	patchData := []patch.PatchOperation{}
+	patchSet := patch.New()
 
 	// reset only if verbosity exists, otherwise do nothing
 	if isReset && len(currentLv) != 0 {
 		if !hasDeveloperConfiguration {
 			// if DeveloperConfiguration is absent, add DeveloperConfiguration first
-			addPatch(&patchData, patchAdd, dcPath, &v1.DeveloperConfiguration{})
+			patchSet.AddOption(patch.WithAdd(dcPath, v1.DeveloperConfiguration{}))
```

Comment:
nit for consistency:
Here we use the object, while in line 305 we use the pointer. Just to share knowledge in case someone will touch it in the future, we can uniform this.